.dotnet/src/Custom/FineTuning/FineTuningJobOperation.Protocol.cs

```diff
@@ -14,25 +15,33 @@ namespace OpenAI.FineTuning;
 /// A long-running operation for creating a new model from a given dataset.
 /// </summary>
 [Experimental("OPENAI001")]
-public class FineTuningJobOperation : OperationResult
+public partial class FineTuningJobOperation : OperationResult
 {
-    private readonly ClientPipeline _pipeline;
-    private readonly Uri _endpoint;
+    internal readonly ClientPipeline _pipeline;
+    internal readonly Uri _endpoint;
+    internal readonly string _jobId;
 
-    private readonly string _jobId;
+
+    new public bool HasCompleted
```

Comment:
I might have missed this in the chat, but does this mean the user needs to call UpdateStatus to get a current bool value here? Would it be better to make a call here if it hasn't been updated in a while?